source/includes/trip-api.md

```diff
@@ -895,6 +927,81 @@ curl https://api.staging.mydaytrip.net/partners/v3/trip/details/bookingId
 }
 ```
 
+> Example response without meeting and drop off positions:
+
+```json
+{
+   "bookingReference": "72R537",
+   "status": "Confirmed",
+   "createdAt": "2022-12-05T18:00:00Z",
+   "passengersCount": 3,
+   "currency": "EUR",
+   "meetingPosition": {
+      "lat": 46.067648,
+      "lon": 7.775185,
+      "description": "the train station in Tasch",
```

Comment:
We don't return googleAdress as well?

Reply:
I will ask, for the meeting positions I checked it was not set so it would have to be optional.

Reply:
While setting restricted areas on FE you need to add googleAdress, otherwise it cannot be saved

Reply:
For some positions inside the airports for example I don't think it should be mandatory.

I think if partner wants to show the map it's better to use the coordinates. The optional Google address might add confusion.

Reply:
You can still adjust coordinates after filling in google address for precise pin

Reply:
Oh you mean about the partner, this is tricky thing.

Reply:
Yeah, if coordinates can be adjusted without changing the Google address then we should definitely return just the coordinates.